npm/azure-npm.yaml

```diff
@@ -146,7 +146,7 @@ metadata:
 data:
   azure-npm.json: |
     {
-        "ResyncPeriodInMinutes": 15,
```

Comment:
It is recommended to have the resync, unless client go and our controller is completely bug free. Resync is kind of a insurance for us, it does good for us than any harm (except for some CPU cycles?)